src/plugins/console/public/application/containers/editor/legacy/console_editor/editor_output.tsx

```diff
@@ -55,6 +56,63 @@ function modeForContentType(contentType?: string) {
   return 'ace/mode/text';
 }
 
+interface Output {
+  [key: string]: {};
+}
+
+function parseResult(response: VectorTile) {
```

Comment:
Can we make this function name more specific? As it reads, I could easily interpret this as a generic function for parsing _any_ result. For example, `convertMapboxVectorTileToJson` would make the purpose of this function super-clear to me.